main.sh

```diff
@@ -56,7 +67,7 @@ function _fmt ()      {
   fi
 
   local color_reset="\x1b[0m"
-  if [[ "${TERM}" != "xterm"* ]] || [ -t 1 ]; then
+  if [[ "${TERM:-}" != "xterm"* ]] || [ -t 1 ]; then
```

Comment:
Allow some variables to be unset, without causing errors, mostly those provided by the system and built-ins that should be set in most reasonable environments, but if not, they won't cause problems.